android/app/src/androidTest/kotlin/coop/polypoly/polypod/CommunicationThroughPodApiWorks.kt

```diff
@@ -67,11 +67,11 @@ class CommunicationThroughPodApiWorks {
     @get:Rule
     val activityRule = ActivityScenarioRule(MainActivity::class.java)
 
-    @Test
+    // TODO: fetch instrumentation tests are disabled for now
```

Comment:
Still the `@Ignore` thing!

Reply:
Can you clarify this? I think that what you mean is that instead of using `TODO` an Android directive should be used, `@ignore`, but just in case.